.github/workflows/copy-workflow.yml

```diff
@@ -105,11 +120,9 @@ jobs:
         CONTEXT: |
           {
             "config": ${{ toJSON(matrix.cfg) }},
-            "github": {
-              "default_branch": "${{ env.DEFAULT_BRANCH }}"
-            }
+            "github": ${{ toJSON(fromJSON(steps.github.outputs.json)) }}
           }
-        TEMPLATE_ENGINE: s#\$\{\{\{\s*(.*?)\s*\}\}\}#`jq -cjn 'env.CONTEXT | fromjson.$1'`#ge
+        TEMPLATE_ENGINE: s#\$\{\{\{\s*\.?(.*?)\s*\}\}\}#`jq -cjn 'env.CONTEXT | fromjson | .$1'`#ge
```

Comment:
we're making the `.` optional then normalize the output so that it always starts with a `.`, correct?

Reply:
Yeah. I felt that when you end up using full blown jq in the template, it looked weird without the leading dot. It's purely about aesthetics.